docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
     environment:
       HASURA_GRAPHQL_ADMIN_SECRET: $HASURA_GRAPHQL_ADMIN_SECRET
       HASURA_GRAPHQL_DATABASE_URL: postgres://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB
-      HASURA_GRAPHQL_ENABLE_CONSOLE: "false" # Must be run manually to track migrations
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true" # Must be run manually to track migrations
```

Comment:
this feels a little dangerous for production security wise if anyone can open the console, even without the secret, i think it's possible to configure local Hasura CLI to point to production though and connect to the production deployment in console, maybe that would be a better option

Reply:
https://hasura.io/docs/2.0/deployment/production-checklist/#disable-console